index.rst

```diff
@@ -16,6 +16,27 @@ metadata counterpart of miniSEED is `StationXML <http://docs.fdsn.org/projects/s
 which is used to describe characteristics needed to interpret the data
 such as location, instrument response, etc.
 
+----------
+Versioning
+----------
+
+Starting with version 3 of **miniSEED**, the specification version is a single
+integer. Any non-backward compatible change to the structure of the header or
+record results in an increment to this number. There are no minor revisions.
```

Comment:
Should we also indicate that there is a direct 1:1 correlation with the _Format version_ field of records that are written to be compliant with a given specification version? If there is agreement with this, then I've made a wording suggestion below.

Reply:
```suggestion
record results in an increment to this number. The specification version
corresponds to the Format version identified in records written to conform
with this version of the specification. There are no minor revisions.
```